src/router/routes/index.ts

```diff
@@ -0,0 +1,14 @@
+import HomeView from '@/views/HomeView.vue'
+import type { RouteRecordRaw } from 'vue-router'
+import modules from './modules'
+
+const routes: RouteRecordRaw[] = [
+  {
+    path: '/',
+    name: 'home',
+    component: HomeView
+  },
+  ...modules
```

Comment:
We don't have a handler for unknown routes (404 error page), so if you think is valid please add it.